backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -60,4 +60,8 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 	public Map<UUID, Shop> getAllShops() {
 		return shopMap;
 	}
+
+	public List<Token> getTokensByShopId(UUID shopId) {
```

Comment:
Later we may want to add another parameter: shopFilter. Keep this in mind. Will discuss with you later today.